backend/controllers/logController.js

```diff
@@ -55,36 +56,29 @@ logController.queryLog = () => {
   }
 };
 
-logController.formatLog = logList => (
-  logArrayConverter(logList)
-);
+logController.formatLog = (logList) => logArrayConverter(logList);
 
 logController.saveLog = async (log) => {
   log.shift();
   try {
     const allLogs = [];
-    const logPromises = log.map(([
-      namespace,
-      lastSeen,
-      type,
-      reason,
-      object,
-      message
-    ]) => {
-      const createdAt = new Date().toISOString();
-      const newEntry = {
-        namespace,
-        lastSeen,
-        type,
-        reason,
-        object,
-        message,
-        createdAt
-      };
-      allLogs.push(newEntry);
-    });
-
-    storage.set('logs', {data: allLogs}, (err) => {
+    const logPromises = log.map(
```

Comment:
Not to be fixed in this PR but to keep in mind for our call, but we're not returning logPromises anymore, so we should change `map` to `forEach` and not save this work into a variable.